.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,49 @@
+# SPDX-FileCopyrightText: Copyright 2021 M2mobi B.V., Amsterdam, The Netherlands
+# SPDX-FileCopyrightText: Copyright 2022 Move Agency Group B.V., Zwolle, The Netherlands
+# SPDX-License-Identifier: CC0-1.0
+
+name: Integration Tests
+on: [push, pull_request]
+
+jobs:
+  phpunit:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
+    name: "PHP-${{ matrix.php-versions }}: Integration"
+    strategy:
+      matrix:
+        php-versions: ['8.3']
+        experimental: [false]
+        include:
+          - php-versions: 8.4
+            experimental: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+          extensions: uopz
+          tools: phpunit:9.5.x
+
+      - name: Setup problem matchers for PHP
+        run: echo "::add-matcher::${{ runner.tool_cache }}/php.json"
+
+      - name: Install dependencies
+        run: composer update
+
+      - name: Install JSONSchema
+        run: pipx install check-jsonschema
+
+      - name: Check payload
+        run: |
+          for type in "message"; do
+            echo "Generate:"
+            php ./get_payload.php "$type" | tee "payload.$type.json"
+
+            echo "\nValidate:"
+            check-jsonschema --schemafile "doc/schemas/$type.schema.json" "payload.$type.json"
```

Comment:
If we set static data why not validate against a real json file? We normally use the json schema for dynamic data but it looks here like the values of the output do not change